python/shotgun_api3/lib/httplib2/__init__.py

```diff
@@ -1,1514 +0,0 @@
-from __future__ import generators
```

Comment:
Everything from here on out is shotgun api being removed, so no need to scroll further down.